src/components/VInput/VInput.vue

```diff
@@ -49,7 +49,7 @@
         :checked="localValue === undefined ? $attrs.checked : localValue"
         v-bind="bind"
         @change="localValue = $event.target.checked"
-        @blur.once.once="dirty = true"
+        @blur.once="dirty = true"
```

Comment:
It's funny. I think this must be getting added by some auto formatter because I swear I've removed it like 3 times now. That or Im going crazy.